data/transit/route/bus.json

```
@@ -2854,7 +2854,9 @@
       "locationSet": {"include": ["us"]},
```

```suggestion
      "locationSet": {"include": ["us-ca.geojson"]},
```
This should be restricted to just California